implementing-nosto/implement-search/implement-search-using-code-editor/implementing-search-page.md

```diff
@@ -438,7 +460,24 @@ init({
 })
 ```
 
-Once fallback is enabled, if the search request fails to retrieve data, the search functionality will be temporarily disabled for a short period, and the user will be redirected to the same page they are currently on.
+Once fallback is enabled, if the search request fails to retrieve data, the search functionality will be temporarily disabled for a short period, and the native content Nosto has overridden will be restored.
+
+### Alternative Fallback Behaviour
+
+If the behaviour described above is undesirable, the configuration supports an alternative option. Fallback mode can be set to `fallback: 'legacy'`, in which case the user will see a page reload if the search request fails.
```

Comment:
_in which case the user will see a page reload if the search request fails_

Seems incomplete. Can you please explain what gets rendered after reload?

Reply:
Also, explain all the new configuration properties introduced and it's possible values at the beginning of the documentation. Here we are explaining `fallback: legacy` which could raise questions among the users